src/tests/interactions-utils.ts

```diff
@@ -116,19 +116,19 @@ export const interactionSuite =
     }
   };
 
-export const getByTestId = (rootElement: HTMLElement, dataTestId: string) => {
+export const getByTestId = (rootElement: HTMLElement | BoundFunctions<typeof queries>, dataTestId: string) => {
```

Comment:
what's `BoundFunctions` and why do you need it?

Reply:
BoundFunctions is the type of the Canvas param in the storybook interaction tests, this is a fix for being able to use the function in these files in TS code